web/src/lib/components/layouts/user-page-layout.svelte

```diff
@@ -1,12 +1,28 @@
 <script lang="ts">
   import { openFileUploadDialog } from '$lib/utils/file-uploader';
-  import type { UserResponseDto } from '@api';
   import NavigationBar from '../shared-components/navigation-bar/navigation-bar.svelte';
   import SideBar from '../shared-components/side-bar/side-bar.svelte';
+  import EyeOutline from 'svelte-material-icons/EyeOutline.svelte';
+  import Close from 'svelte-material-icons/Close.svelte';
+  import IconButton from '../elements/buttons/icon-button.svelte';
+  import { fly } from 'svelte/transition';
+  import type { UserResponseDto } from '@api';
+  import { createEventDispatcher } from 'svelte';
+  import { quintOut } from 'svelte/easing';
+  import CircleIconButton from '../elements/buttons/circle-icon-button.svelte';
+
   export let user: UserResponseDto;
   export let hideNavbar = false;
   export let showUploadButton = false;
   export let title: string | undefined = undefined;
+  export let selectHidden = false;
+  export let countTotalPeople: number | undefined = undefined;
```

Comment:
I still don't understand why this generic "user page layout" needs specific stuff about "people" in it. These variables and logic belong in the associated show/hide person page.

Reply:
This is for the `Show & hide faces` button, we show it only in the /people tab